templates/module/src/entity-content-route-provider.php.twig

```diff
@@ -161,7 +161,7 @@ class {{ entity_class }}HtmlRouteProvider extends AdminHtmlRouteProvider {% endb
       $route = new Route("/admin/structure/{$entity_type->id()}/settings");
       $route
         ->setDefaults([
-          '_form' => 'Drupal\{{ module }}\Controller\{{ entity_class }}SettingsForm',
+          '_form' => 'Drupal\{{ module }}\Form\{{ entity_class }}SettingsForm',
```

Comment:
Ah!  That makes perfect sense - yes I was testing without a bundle based entity.  Thanks for explaining :)